autocertkit/operations_tests.py

```diff
@@ -233,42 +233,37 @@ def test_crashdump(self, session):
         """Check crashdump is created properly."""
         log.debug("Running Crashdump test.")
         if not get_reboot_flag():
-            numberofcds = len(retrieve_crashdumps(session))
             tc_info = {}
             tc_info['device'] = self.config['device_config']['udid']
             tc_info['test_class'] = self.__class__.__module__ + '.' + self.__class__.__name__
             tc_info['test_method'] = 'test_crashdump'
-            tc_info['cds'] = numberofcds
+            tc_info['existing_crashdumps'] = [cd['timestamp'].strftime("%Y-%m-%d %H:%M:%S") for cd in retrieve_crashdumps(session)]
+            tc_info['crash_begin_time'] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
             set_reboot_flag(tc_info)
-            crashtime = get_reboot_flag_timestamp()
-            log.debug("The reboot flag's timestamp is set to: '%s'" % str(crashtime))
+            log.debug("The reboot flag's timestamp is set to: '%s'" % str(get_reboot_flag_timestamp()))
             time.sleep(5) # host crash need to be done after flag is created to compare.
             host_crash(self.session)
-
-        # Check number of crashdumps are increased by 1.
+        # Check tc_info is persistent.
         tc_info = get_reboot_flag()
-        numberofcds = len(retrieve_crashdumps(session))
-        if not 'cds' in tc_info:
-            raise Exception("Reboot flag does not include crashdump info. Does host restarted by forced crashdump?")
-
-        if not numberofcds == tc_info['cds'] + 1:
-            raise Exception("Host does not created crashdump properly: expected %d, found %d" %
-                    (tc_info['cds'] + 1, numberofcds))
-        
+        log.debug("tc_info from reboot flag: %s" % str(tc_info))
+        if 'existing_crashdumps' not in tc_info or 'crash_begin_time' not in tc_info:
+            raise Exception("Reboot flag is not persistent and does not include crashdump info. Does host restarted by forced crashdump?")
+        crash_beg_time = datetime(*(time.strptime(tc_info['crash_begin_time'],"%Y-%m-%d %H:%M:%S")[0:6]))
```

Comment:
Right, I removed them thinking coverage coveralls is failing because of them but it looks I am wrong. I will add CR/LF along with changes from first review comment.